gravitee-apim-console-webui/src/management/api/entrypoints-v4/edit/api-entrypoints-v4-edit.component.html

```diff
@@ -23,7 +23,30 @@
   <form [formGroup]="form" *ngIf="form; else loader" (ngSubmit)="onSaveEntrypointConfig()">
     <div *ngIf="entrypointSchema" class="api-creation-v4__form-container">
       <h3>{{ entrypointName }}</h3>
-      <gio-form-json-schema [jsonSchema]="entrypointSchema" [formControlName]="entrypoint.type"></gio-form-json-schema>
+
+      <mat-card class="qos">
+        <gio-banner-info>
+          Quality of service defines the guaranteed level of message delivery.
+          <div gioBannerBody>
+            A given quality of service may or may not be supported by a given endpoint. Support also depends on the protocol used for the
+            entrypoint. Please check the documentation for QoS compatibility.
+            <a
+              href="https://documentation.gravitee.io/apim/guides/api-configuration/v4-api-configuration/quality-of-service"
+              aria-label="learn more"
+              target="_blank"
+              >Learn more</a
+            >
+          </div>
+        </gio-banner-info>
+        <mat-form-field class="qos__select">
+          <mat-label>Quality of Service</mat-label>
+          <mat-select aria-label="Quality of service" [formControlName]="entrypoint.type + '-qos'" required>
+            <mat-option *ngFor="let supportedQos of supportedQos" [value]="supportedQos">{{ supportedQos }}</mat-option>
+          </mat-select>
+        </mat-form-field>
+      </mat-card>
+
+      <gio-form-json-schema [jsonSchema]="entrypointSchema" [formControlName]="entrypoint.type + '-config'"></gio-form-json-schema>
```

Comment:
thought: it feels strange to have the configuration after the QoS. I would have kept the gio-form-json-schema before the QoS section

Reply:
From my point of view, it's not strange 😄. But we can check with @anthonyjgatti and @audrey-tlm

Reply:
Feedback from Audrey, yes it should be at after the configuraiton form.
I will do it